src/main/java/com/nike/cerberus/cli/EnvironmentConfigToArgsMapper.java

```diff
@@ -120,8 +121,9 @@ private EnvironmentConfigToArgsMapper() {
             case UploadCertificateFilesCommand.COMMAND_NAME:
                 args = getUploadCertFilesCommandArgs(environmentConfig, passedArgs);
                 break;
+            case CreateCmsAsgCommand.COMMAND_NAME:
             case CreateCmsClusterCommand.COMMAND_NAME:
-                args = getCreateCmsClusterCommandArgs(environmentConfig, stackRegion);
```

Comment:
Can someone still run create-cms-cluster correctly with env config?

Reply:
Yes if the env config is provided, the following method will grab the necessary args from the config. It's tested in the create-cluster unit test. I also ran the command with an env config and with only parameters to check.